modules/rosa-hcp-sts-creating-a-break-glass-cred-cli.adoc

```diff
@@ -0,0 +1,138 @@
++
+[source,terminal]
+----
+$ rosa create break-glass-credential -c <cluster_name> -i <1>
+----
+<1> Replace <cluster_name> with the name of your cluster.
++
+.Example output
++
+[source,terminal]
+----
+I: Enabling interactive mode
+? Username (optional): <1>
+? Expiration duration (optional): <2>
+I: Successfully created a break glass credential for cluster 'ac-hcp-test'.
+----
+<1> If left blank, the username defaults to your username.
+<2> The minimum validity of the break glass credential is 10 minutes, and the maximum validity is 24 hours. If left blank, the expiration duration defaults to 24 hours.
++
+** Create a break glass credential for cluster called `mycluster` with specified values:
++
+[source,terminal]
+----
+$ rosa create break-glass-credential -c mycluster --username test-username --expiration 1h
+----
++
+. List the break glass credential IDs, status, and associated user that is available for a cluster called `mycluster` by running the following command:
++
+[source,terminal]
+----
+$ rosa list break-glass-credential -c mycluster
+----
++
+.Example output
++
+[source,terminal]
+----
+ID                                USERNAME    STATUS
+2a7jli9n4phe6c02ul7ti91djtv2o51d  test-user   issued
+----
++
+[NOTE]
+====
+You can also view the credentials in a JSON output by adding the `-o json` argument to the command.
+====
+
+. View information about the break glass credential with one of the following commands:
++
+** To view the status of the break glass credential, run the following command, replacing <break_glass_credential_id> with the break glass credential ID:
++
+[source,terminal]
+----
+$ rosa describe break-glass-credential <break_glass_credential_id> -c <cluster_name>
+----
++
+.Example output
++
+[source,terminal]
+----
+ID:                                    2a7jli9n4phe6c02ul7ti91djtv2o51d
+Username:                              test-user
+Expire at:                             Dec 28 2026 10:23:05 EDT
+Status:                                issued
+----
++
+The following is a list of possible `Status` field values:
++
+* `issued` The break glass credential has been issued and is ready to use.
+* `expired` The break glass credential has expired and can no longer be used.
+* `failed` The break glass credential has failed to create. In this case, you will receive a service log detailing the failure. For more information about service logs, see _Accessing the service logs for Red Hat OpenShift Service on AWS clusters_. For steps to contact Red Hat Support for assistance, see _Getting support for Red Hat OpenShift Service on AWS_.
+* `awaiting_revocation` The break glass credential is being revoked, meaning it cannot be used.
+* `revoked` The break glass credential has been revoked and can no longer be used.
++
+** Use the following command to retrieve the `kubeconfig`:
++
+[source,terminal]
+----
+$ rosa describe break-glass-credential <break_glass_credential_id> -c mycluster --kubeconfig
+----
+.Example output
++
```

Comment:
```suggestion

```

Reply:
I need the '+' here otherwise the example output does not indent correctly.